server/render/test/index.test.js

```diff
@@ -0,0 +1,232 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { shouldServerSideRender, setShouldServerSideRender } from '..';
+import { setShouldServerSideRenderLogin } from '../../../client/login/ssr';
+
+let mockReturnValues = {};
+
+jest.mock( 'lib/i18n-utils', () => {
+	return {
+		isDefaultLocale: () => mockReturnValues.isDefaultLocale,
```

Comment:
Hmm, why not just `=> undefined`? (Same for the instances below.) I find the detour through `mockReturnValues` a bit misleading here...

Reply:
Ohh, it took me a while to find `remock` https://github.com/Automattic/wp-calypso/pull/25554/files#diff-be817af57eab90eff3023a01926ac711R49

Hmm, still a bit confusing. Wonder if we have some established practices for patterns like that...

Reply:
I have:
* moved `function remock` closer to `mockReturnValues`, just to express clearly the connection
* in the initial version I wanted to add const defaultMockContext = ... and have thew `mockReturnValues` assigned to this (to avoid the possible undefined confusion), but in the end I just put them in remock without creating a new const.
* added a DocBlock explaining a bit the behavior

I went in this direction just because by trying `jest.mock` inside a test didn't seem to work/override the global state. My intention is pretty simple: I just want to create a mock of those methods, and have them return a specific value for each test (preferably without writing tens of line of code in each of the tests)